dotcom-rendering/docs/snyk/how-to.md

```diff
@@ -0,0 +1,21 @@
+## How to run Snyk locally?
+
+1. Install the Snyk CLI
+   * using npm: `npm install snyk -g`
+   * using yarn: `yarn global add snyk`
+2. Run `snyk auth`
```

Comment:
Might be helpful if there's any information about _how_ to log in through snyk (e.g guardian uses google accounts?) or even info on how to get access (ask your teams em)? I'm not sure I confidently know the answers so it'd be great to have it written down!

Reply:
Good point - you might add it as a requirement to this `how-to` - you need either the EM, or if they're not about, DevX to add you to the `guardian-dotcom-n2y` org (part of the Guardian group).

`snyk auth` uses a browser popup and we SSO to login.

There is also the [official doc](https://docs.snyk.io/snyk-cli/authenticate-the-cli-with-your-account) with a few ways to do this.

Reply:
Good idea! Added the info